paint-timing/fcp-only/fcp-gradient.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP should not fire with gradient-only backgrounds</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+        background-image: linear-gradient(to bottom, orange, blue);
+    }
+
+    .contentful {
```

Comment:
Perhaps comment that this is used to be able to use test_fcp(). Alternatively, we can have test_no_fcp() or even just use assertNoFirstContentfulPaint directly.

Reply:
ok